proteuslib/tools/parameter_sweep.py

```diff
@@ -438,6 +469,12 @@ def parameter_sweep(model, sweep_params, outputs, results_file=None, optimize_fu
     if rank == 0 and results_file is not None:
         # Save the global data
         np.savetxt(results_file, global_save_data, header=data_header, delimiter=',', fmt='%.6e')
+
+        if interpolate_nan_outputs:
+            global_results_clean = _interp_nan_values(global_values, global_results)
+            global_save_data_clean = np.hstack((global_values, global_results_clean))
+
+            np.savetxt('%s_clean' % (results_file), global_save_data_clean, header=data_header, delimiter=',', fmt='%.6e')
```

Comment:
```suggestion
            np.savetxt('interpolated_%s' % (results_file), global_save_data_clean, header=data_header, delimiter=',', fmt='%.6e')
```
We should be specific with this name

Reply:
yep, interpolated is definitely the clearer name here, good catch